MapifyEditor/Export/Validators/Project/MapInfoValidator.cs

```diff
@@ -42,6 +42,18 @@ protected override IEnumerator<Result> Validate(Scenes scenes)
                 yield return Result.Error($"The spawn position's Y value must be above the terrain ({worldHeight})", mapInfo);
             if (spawnPos.y < mapInfo.waterLevel)
                 yield return Result.Error($"The spawn position must be above the water level ({mapInfo.waterLevel}", mapInfo);
+
+            if (mapInfo.useFixedMapImage)
+            {
+                if (mapInfo.fixedMapImage == null)
+                {
+                    yield return Result.Error("Mapinfo 'fixedMapImage' must be set when 'useFixedMapImage' is true", mapInfo);
```

Comment:
Please use `nameof` for class/variable names

Reply:
fixed